src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/PojoTest.java

```diff
@@ -28,11 +28,12 @@
         Token.class,
         UserInfo.class,
         HistoryVariableInstance.class,
-        AddLocalVariableRequest.class
+        AddLocalVariableRequest.class,
```

Comment:
I didn't know Java allowed you to do that will stop this class always conflicting if two people add classes to it at the same time